src/components/VContentReport/VContentReportForm.vue

```diff
@@ -1,180 +1,186 @@
+  <div id="content-report-form" class="w-80 p-6">
+    <div v-if="status === statuses.SENT">
+      <p class="font-semibold text-2xl mb-4">
+        {{ $t('media-details.content-report.success.title') }}
       </p>
-
-      <VButton
-        :disabled="!reasonSelected"
-        variant="secondary"
-        class="float-end bg-trans-blue"
-        @click="onIssueSelected"
+      <i18n
+        path="media-details.content-report.success.note"
+        class="text-sm"
+        tag="p"
       >
-        {{ $t('photo-details.content-report.next') }}
-      </VButton>
-    </form>
+        <template #source>
+          <a
+            :href="media.url"
+            class="text-pink hover:underline"
+            target="_blank"
+            rel="noopener"
+            >{{ providerName }}</a
+          >
+        </template>
+      </i18n>
+    </div>
+
+    <div v-else-if="status === statuses.FAILED">
+      <p class="font-semibold text-2xl mb-4">
+        {{ $t('media-details.content-report.failure.title') }}
+      </p>
+      <p class="text-sm">
+        {{ $t('media-details.content-report.failure.note') }}
+      </p>
+    </div>
+
+    <!-- Main form -->
+    <div v-else>
+      <div class="font-semibold text-2xl mb-4">
+        {{ $t('media-details.content-report.long') }}
+      </div>
+
+      <p class="text-sm mb-4">
+        {{ $t('media-details.content-report.form.disclaimer') }}
+      </p>
+
+      <form class="text-sm">
+        <fieldset class="mb-4 flex flex-col">
+          <legend class="font-semibold mb-4">
+            {{ $t('media-details.content-report.form.question') }}
+          </legend>
+          <template v-for="reason in reasons">
+            <VRadio
+              :id="reason"
+              :key="reason"
+              v-model="selectedReason"
+              class="mb-4"
+              name="reason"
+              :value="reason"
+            >
+              <div class="flex flex-col gap-2 w-full">
+                <div>
+                  {{ $t(`media-details.content-report.form.${reason}.option`) }}
+                </div>
+                <VDmcaNotice
+                  v-if="reason === reasons.DMCA"
+                  v-show="reason === selectedReason"
+                  :provider="providerName"
+                  :media-url="media.foreign_landing_url"
+                />
+                <VOtherIssueForm
+                  v-if="reason === reasons.OTHER"
+                  v-show="reason === selectedReason"
+                  key="other"
+                  v-model="description"
+                />
+              </div>
+            </VRadio>
+          </template>
+        </fieldset>
+
+        <div class="flex flex-row items-center justify-end gap-4">
+          <VButton variant="tertiary" @click="handleCancel">
+            {{ $t('media-details.content-report.form.cancel') }}
+          </VButton>
+
+          <VButton
+            v-if="selectedReason === reasons.DMCA"
+            key="dmca"
```

Comment:
If both `v-if` and `v-else` have the same component, Vue will try to keep the same one and just change the props. This causes a problem here because one of the buttons is dynamically changed to an anchor link. Using different keys forces Vue to actually destroy and recreate the component.

Reply:
Ahhh I see!! Totally makes sense, thanks for explaining 🙏